src/dataExplorer/components/resources/types/script/index.ts

```diff
@@ -17,10 +22,18 @@ export default function script(register) {
     editor,
     init: id => {
       if (!id || !CLOUD) {
+        const language = getLanguage()
```

Comment:
Does this `init` function happens when the `TemplatePage` is being called? Just trying to understand how `getLanguage()` get the language type from the URL